companies/igs-goe.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "igs-goe",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "public body",
+        "school"
+    ],
+    "name": "Georg-Christoph-Lichtenberg-Gesamtschule Göttingen-Geismar",
+    "address": "Schulweg 22\n37083 Göttingen",
```

Comment:
Please add the country (`Deutschland`) to the address.

Reply:
Done, thanks!